Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1029,7 +1029,12 @@ internal static void SpawnObject(GameObject obj, NetworkConnection ownerConnecti
                 return;
             }
             identity.Reset();
-            identity.connectionToClient = (NetworkConnectionToClient)ownerConnection;
+
+            if (ownerConnection != null)
+            {
+                identity.connectionToClient = (NetworkConnectionToClient)ownerConnection;
```

Comment:
move this line before the if,  

if ownerconnection is null,  you want to set identity.connectionToClient to null.   This can happen when the object loses authority

Reply:
identity.Reset(); sets it null already right above